lib/datalad_data_provider.rb

```diff
@@ -46,51 +55,25 @@ def is_fast_syncing? #:nodoc:
     false
   end
 
-  def provider_full_path(userfile) #:nodoc:
-    datalad_relative_path(userfile) + userfile.name
+  def provider_full_path(userfile) #:nodoc: # not sure is needed anymore
```

Comment:
Normally, this method is used both as a way to provide a representation of a file's location on the remote site, but also it is supposed to be called internally to build a 'path' to the file. Setting it so that it returns something like "#{relative_path}/#{userfile.name}" would be useful everywhere else in the code where this is done right now.

Reply:
Are you saying it is needed or are you saying I did this wrong?